### PR TITLE
Search border enhancement and Enter bug

### PR DESCRIPTION
**Pull request by jshi22**
Enter button now properly adds courses. Top border removed and border now changes when search results show, just like chrome.

### CODE REVIEW

**Comment by alihacks** [comment]
Looks like you have some conflicts. See if you can rebase your changes from master it just use a new branch from master.

File: static/js/redux/ui/search_bar.jsx [hunk 1]
```diff
@@ -143,36 +143,42 @@ class SearchBar extends React.Component {
     const currSem = ($(window).width() < 767) ?
             SearchBar.getAbbreviatedSemesterName(this.props.semester) :
             SearchBar.getSemesterName(this.props.semester);
+    const transformSearchAppearance = this.props.searchResults.length > 0 && this.state.focused;
     return (
       <div className="search-bar no-print">
-        <div className="search-bar__wrapper">
+        <div className="search-bar__wrapper" style={{ borderRadius: transformSearchAppearance ? '10px 10px 0px 0px' : '25px 25px 25px 25px',
```
Comment (kristinyim) [inline static/js/redux/ui/search_bar.jsx]: instead of having the '10px ...' in here, lets make them classes

Comment (kristinyim) [inline static/js/redux/ui/search_bar.jsx]: lets make these classes instead of explicitly having the css in here

**Comment by kristinyim** [comment]
Made it cleaner using a single class that'll be applied upon dropdown.
Adjusted bar.
Mobile friendly:
image
image

**Pull request merged**